pkg/webhook/webhook.go

```diff
@@ -223,6 +309,25 @@ func getServiceAccountTokenExpiration(pod *corev1.Pod, sa *corev1.ServiceAccount
 	return serviceAccountTokenExpiration, nil
 }
 
+// getProxyPort returns the port for the proxy init container and the proxy sidecar container
+func getProxyPort(pod *corev1.Pod) (int32, error) {
+	if pod.Annotations == nil {
```

Comment:
should we just check `len(pod.Annotations) == 0` similar to `shouldInjectProxySidecar`?